lib/rtcpeerconnection/chrome.js

```diff
@@ -117,12 +124,41 @@ function ChromeRTCPeerConnection(configuration) {
 
 inherits(ChromeRTCPeerConnection, EventTarget);
 
-ChromeRTCPeerConnection.prototype.addStream = function addStream(stream) {
-  if (this._localStreams.has(stream)) {
-    return;
```

Comment:
Any harm to leaving this as-is?

Reply:
@markandrus Why would we need to retain this any longer?

Reply:
I guess you're right 👍